doc/releases/release-notes-2.6.rst

```diff
@@ -43,6 +43,14 @@ Deprecated in this release
   :c:macro:`DT_INST_PWMS_LABEL` was deprecated in favor of utilizing
   :c:macro:`DT_PWMS_CTLR` and variants.
 
+* :c:macro:`DT_IO_CHANNELS_LABEL_BY_IDX`,
+  :c:macro:`DT_IO_CHANNELS_LABEL_BY_NAME`,
+  :c:macro:`DT_IO_CHANNELS_LABEL`,
+  :c:macro:`DT_INST_IO_CHANNELS_LABEL_BY_IDX`,
+  :c:macro:`DT_INST_IO_CHANNELS_LABEL_BY_NAME`, and
+  :c:macro:`DT_INST_IO_CHANNELS_LABEL` was deprecated in favor of utilizing
```

Comment:
I'm just noticing this, but this should be "were deprecated" instead of "was deprecated". I see this just being copy/pasted but we might as well not add more.